README.md

```diff
@@ -79,6 +79,16 @@ For more commands type:
 > help
 ```
 
+## Installing libdeltachat system wide
+
+```
+$ git clone https://github.com/deltachat/deltachat-core-rust.git
+$ cd deltachat-core-rust
+$ cmake -B build .
```

Comment:
Just to make sure: This works without creating `build/` beforehand?

Reply:
Yes